sdk/openai/Azure.AI.OpenAI/src/Custom/ChatCompletions/ChatCompletionsOptions.Serialization.cs

```diff
@@ -130,6 +130,30 @@ void IUtf8JsonSerializable.Write(Utf8JsonWriter writer)
             writer.WritePropertyName("seed"u8);
             writer.WriteNumberValue(Seed.Value);
         }
+        if (Optional.IsDefined(EnableLogProbabilities))
+        {
+            if (EnableLogProbabilities != null)
+            {
+                writer.WritePropertyName("logprobs"u8);
+                writer.WriteBooleanValue(EnableLogProbabilities.Value);
+            }
+            else
```

Comment:
Is `WriteNull()` required for serialization to work properly? Otherwise, I'd suggest to take it away to align with the rest serialization code.

Reply:
Probably?

I lifted that section verbatim from the generated code:

https://github.com/ninghu/azure-sdk-for-net/blob/ba4468e1ebd93d70803d1159a1c42aa66445c19b/sdk/openai/Azure.AI.OpenAI/src/Generated/ChatCompletionsOptions.Serialization.cs#L130-L153

Those two properties seemed to be the only ones that explicitly wrote null.

Reply:
As long as other ChatCompletion tests don't fail, we are good.

Reply:
After searching `WriteNull()` throughout the solution, it appears that it's only used in generated code and not in any custom code. It seems safe to remove it.

Reply:
Resolved by https://github.com/ninghu/azure-sdk-for-net/pull/15/commits/750b4e319b2c7083c8287d78129adfa2f513d0dd